src/table/virtualized-table/hooks/use-heights.ts

```diff
@@ -41,6 +42,14 @@ const useHeights = ({ columns, columnWidths, pageInfo, headerRef, totalsRef, tot
     boldText: true,
   });
 
+  const resizeAllHeaderCells = headerHeight.resizeAllCells;
+  const resizeAllTotalCells = totalsHeight.resizeAllCells;
+
+  const resizeCells = useCallback(() => {
+    resizeAllHeaderCells();
+    resizeAllTotalCells();
+  }, [resizeAllHeaderCells, resizeAllTotalCells]);
```

Comment:
These dependencies are functions, right? 
My questions is, why this useCallback should depend on another useCallback(ed) function, which does not return anything in result, just changes some internal states of `use-dynamic-row-height`?

If those functions are already in memoized in their own useCallback, what is the point of using useCallback here once more? and is using useCallback here, affects the performance?

Reply:
Not that much point in it to be honest. If I don't wrap them in a useCallback, I would have to return both functions from this hook.

But thinking of it now. Skipping the useCallback, might actually make it a bit more clear  what going in the consuming component, as I would be "force" to change naming a bit.